### PR TITLE
Add a daily-running GH Action to package Bower on new JS SDK tag

### LINKED ISSUE #7: `stellar-sdk` is not getting packaged nor published for Bower
Great work on packaging the Stellar SDK into a single js-file for front-end development!!

I had two questions:
- Do you intend to regularly update the package? I noticed that the Typescript version is on 9.0.0 now?
- Would you be able to offer something similar on Helium --> https://github.com/helium/helium-js   

Thanks!

### PR DESCRIPTION
**Pull request by Shaptic**
This should close #7.

### CODE REVIEW

File: .github/workflows/repackage.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+          cd sdk
+          yarn install
+          yarn
+          cp dist/* bower/
+
+      - name: Publish package
+        run: |
+          TAG=`git describe --tags`
+          cd bower/
+          [ "$(git log -1 --pretty=%B)" = "$TAG" ] && exit
```
Comment (Shaptic) [inline .github/workflows/repackage.yml]: Explanation: If there's no change between the current latest commit message and the SDK's latest tag, bail. 

This should prevent an empty commit from happening every hour and an early commit happening when the SDK's `master` gets new commits but a release isn't ready yet.

Comment (jacekn) [reply .github/workflows/repackage.yml]: Quick question - is it possible to run the check earlier in the process to save some work? If possible maybe run it before you install node?

Comment (Shaptic) [reply .github/workflows/repackage.yml]: That's a good point :thinking: I think 4c2d6f3 should do it.

File: .github/workflows/repackage.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        run: |
+          cd sdk/
+          TAG=`git describe --tags`
+          cd bower/
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          git add .
+          git commit -m $TAG
+          git tag -a $TAG -m $TAG
+          git push origin master --tags
```
Comment (leighmcculloch) [inline .github/workflows/repackage.yml]: `git push ... --tags` can make for some fun times if we ever have a bug, create a ton of tags, and push them all.

Could we do a push of a specific tag to reduce the blast radius of any future bugs we might have in this area of code? I think the cost is low.

```suggestion
          git push --atomic origin master "$TAG"
```

File: .github/workflows/repackage.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+          repository: stellar/bower-js-stellar-sdk
+          path: sdk/bower
+
+      - name: Check for updated SDK
+        # if this check fails then the job doesn't need to proceed
+        run: |
+          cd sdk/
+          TAG=`git describe --tags`
+          cd bower/
+          "`git log -1 --pretty=%B`" != "$TAG"
```
Comment (leighmcculloch) [inline .github/workflows/repackage.yml:31]: 👀 Fancy.

**Review by leighmcculloch — APPROVED** [review]
One request, otherwise LGTM.

File: .github/workflows/repackage.yml (hunk 2 above)
Comment (Shaptic) [reply .github/workflows/repackage.yml]: Great call!

File: .github/workflows/repackage.yml (hunk 3 above)
Comment (Shaptic) [reply .github/workflows/repackage.yml:31]: Heh, we'll see if it works :thinking:

**Pull request merged**